src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx

```diff
@@ -14,6 +16,9 @@ interface Props {
 export const MyCollectionArtworkSWASectionSubmitted: React.FC<Props> = ({
   artwork,
 }) => {
+  const enablePreConsignment = useFeatureFlag(
+    "onyx_submission_flow_pre_consignment"
```

Comment:
**suggestion**: Very minor but since everything in the flow currently is pre consignment or pre listing, maybe we can name this `post_approval`?

Reply:
Very good idea 👍